Robust.Shared/Localization/LocalizationManager.Entity.cs

```diff
@@ -41,7 +42,7 @@ private bool TryGetEntityLocAttrib(IEntity entity, string attribute, [NotNullWhe
         // Flush caches conservatively on prototype/localization changes.
         private void OnPrototypesReloaded(PrototypesReloadedEventArgs args)
         {
-            if (!args.ByType.TryGetValue(typeof(EntityPrototype), out var changeSet))
+            if (!args.ByType.TryGetValue(typeof(EntityPrototype), out _))
```

Comment:
```suggestion
            if (!args.ByType.ContainsKey(typeof(EntityPrototype)))
```

I know you're cleaning up after my mess here but I can't help but notice.